src/CovidSim.cpp

```diff
@@ -5665,3 +5646,16 @@ int GetInputParameter3(FILE* dat, const char* SItemName, const char* ItemType, v
 	//	fprintf(stderr,"%s\n",SItemName);
 	return FindFlag;
 }
+
+/* helper function to set icdf arrays */
+void setICDF(double* icdf, int startValue)
+{
+	int i;
```

Comment:
Move i declaration inline to for loop.

ie: `for (int i = 0; i < CDF_RES; i++)`

Reply:
Agreed - change committed.